ariadne/cardano/src/Ariadne/MainTemplate.hs

```diff
@@ -5,7 +5,8 @@ module Ariadne.MainTemplate
        , defaultMain
        ) where
 
-import Control.Concurrent.Async (link, race_, withAsync)
+import Control.Concurrent
```

Comment:
Didn't we want to use explicit imports at least for external stuff?

Reply:
Fair enough, as well:)